arcgisUM.py

```diff
@@ -84,7 +84,7 @@ def getArcGISGroupByTitle(arcGISAdmin, title):
     :return: ArcGIS Group object or None
     :rtype: Group or None
     """
-    searchString = "title:"+title
+    searchString = f'title:"{title}"'
     logger.debug("group search string: original: {}".format(searchString))
 
     # quote characters that are special in the group search.
```

Comment:
Did you check whether the translation here still matters? If it's quoted, maybe they aren't interpreted differently.

Reply:
No idea what the translation is doing here. It seems working with the assignment title containing single quotes and double quotes.

Based on the log message, searchString is not changed after the translate statement.

Reply:
The translation is supposed to be escaping meaningful characters ("?" and "*") but because we're quoting the title, I think that may not be necessary anymore. I tested without this line and it worked fine. Should we remove this?

Reply:
If we do remove it, we can probably also remove or modify these debug log messages

Reply:
updated the debug message.